cli/shell/shell.go

```diff
@@ -54,6 +56,42 @@ func Execute(workspace string, command string, args ...string) (string, error) {
 	return strings.Trim(out.String(), "\n"), nil
 }
 
+// GetEnv returns an environment variable as string
+func GetEnv(envVar string, defaultValue string) string {
+	if value, exists := os.LookupEnv(envVar); exists {
+		return value
+	}
+
+	return defaultValue
+}
+
+// GetEnvBool returns an environment variable as boolean
+func GetEnvBool(key string) bool {
+	s := os.Getenv(key)
+	if s == "" {
+		return false
```

Comment:
I kind of wonder if this should return something else here instead of `false`. (Perhaps `nil`?) It seems like this behavior might introduce hard-to-handle error conditions, where a value is incorrectly interpreted as being truly false instead of an error in processing the lookup itself.

Reply:
Mmmm, I see your point: so if the env var is not present, then do not consider it as false... In this particular use case, we are going to check for true, only. If true, then execute the proper code, otherwise use default. But it could be the case that we use this function again checking for false, and if the var is not present, it will satisfy the condition

Reply:
I think returning (bool, error) will do the trick, although we'll have to handle errors on the consume side.

Reply:
I added your suggestion in https://github.com/elastic/e2e-testing/pull/205/commits/3881a99b251b92ee8fd52f05acd8c9e4ed9c14bc, including unit tests for verification.